.github/workflows/test.yml

```diff
@@ -0,0 +1,236 @@
+name: Test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-16.04
+    env:
+      SCCACHE_CACHE_SIZE: "1G"
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - name: Checkout submodules
+        shell: bash
+        run: git submodule update --init --recursive
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+      - name: Cache cargo registry
+        uses: actions/cache@v1.1.2
```

Comment:
Do we need both `actions/cache` and sccache? I thought `sccache` should be enough and it could conflict with cache actions, no?

Reply:
this needs to be checked